_field-types/mapping-parameters/term-vector.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: default
+title: Term vector
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 190
+has_children: false
+has_toc: false
+---
+
+# Term vector
+
+The `term_vector` mapping parameter controls whether term-level information is stored for individual text fields during indexing. This information includes details such as term frequency, position, and character offsets, which can be used for advanced features like custom scoring and highlighting.
+
+By default, `term_vector` is disabled. When enabled, term vectors are stored and can be retrieved using the `_termvectors` API.
```

Comment:
Can we highlight that this API retrieves the term vectors for a particular document?

Reply:
Got it. Please ensure to link to the new documentation here.